Taskfile.yml

```diff
@@ -67,6 +66,23 @@ tasks:
     cmds:
       - podman push ${IMAGE}:${TAG} ${REGISTRY}/${PROJECT}/${IMAGE}:${TAG}
 
+  container-image:deploy:
+    desc: Deploy a locally built container image to a device over SSH
+    vars:
+      SSH_USER: playtron
+    preconditions:
+      - sh: test -n "{{.SSH_USER}}"
+        msg: "A value for SSH_USER must be provided"
+      - sh: test -n "{{.SSH_HOST}}"
+        msg: "A value for SSH_HOST must be provided"
+      - sh: test -n "{{.IMAGE}}"
+        msg: "A value for IMAGE must be provided"
+      - sh: test -n "{{.TAG}}"
+        msg: "A value for TAG must be provided"
+    cmds:
+      - podman save --format oci-archive localhost/{{.IMAGE}}:{{.TAG}} | pv --name "Copying" | ssh {{.SSH_USER}}@{{.SSH_HOST}} 'cat > /tmp/{{.IMAGE}}-{{.TAG}}.oci'
+      - ssh -t {{.SSH_USER}}@{{.SSH_HOST}} 'sudo LANG="en_US.UTF-8" LC_ALL="en_US.UTF-8" bootc switch --transport oci-archive /tmp/{{.IMAGE}}-{{.TAG}}.oci'
```

Comment:
`/var/tmp/` might be a better location since you're not limited to the amount of RAM and/or swap on your system. Some internal builds we're experimenting with right now, for example, are huge.